.circleci/scripts/binary_checkout.sh

```diff
@@ -63,6 +63,8 @@ popd
 # Clone the Builder master repo
 retry git clone -q https://github.com/pytorch/builder.git "$BUILDER_ROOT"
 pushd "$BUILDER_ROOT"
+git fetch
```

Comment:
@malfet @seemethere , this is the only place where I need to point to the correct builder/ branch, right?

Reply:
I suggest we pick the branch explicitly while cloning.

But yes, judging by quick grepping this is the only place